home-page.css

```diff
@@ -0,0 +1,137 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.navigation-bar {
+    display: flex;
+    width: 100%;
+    background-color: rgb(150, 153, 153);
+    justify-content: space-between;
+    padding: 0.9rem;
+}
+.main-container{
+    position: sticky;
+    top: 0;
+    z-index: 1;
+}
+.title {
+    font-size: 2rem;
+    font-weight: 600;
+    margin-left: 1rem;
+    margin-top: 1.2rem;
```

Comment:
You could use the single-line shorthand to achieve the same thing. 
e.g.
`margin: 1.2rem initial initial 1rem`
i.e.
`margin: [top] [right] [bottom] [left]`